app/components/agenda-manager/agenda-context.js

```diff
@@ -180,10 +181,15 @@ export default class AgendaManagerAgendaContextComponent extends Component {
       if(item.position !== index || previousItem !== previous) {
         this.setProperty(item, "position", index);
         this.setProperty(item, "vorigeAgendapunt", previous);
-        const treatment = yield item.treatment;
+        const treatment = yield item.behandeling;
```

Comment:
yikes, does this mean the order of treatments was never correctly saved?

Reply:
it is used here: https://github.com/lblod/notulen-prepublish-service/blob/5e95f3c3601e20b87c05b7f5e85938ce560da9ad/models/treatment.js#L75, so possible incorrect info when an extract is published